tests/raft_scenarios/retire_one

```diff
@@ -58,7 +58,8 @@ dispatch_all
 
 assert_is_primary,0
 assert_is_backup,1
```

Comment:
We could also replace these assert lines (`assert_detail,1,leadership_state,Backup`), if we want to consolidate actions? I don't know if its helpful though.

Reply:
I started doing that, and then noticed the `assert_isnot_*` variants, which can't easily be replaced with `assert_detail`. But now that you mention it, we could easily `assert_!detail`.